vignettes/datatable-programming.Rmd

```diff
@@ -373,6 +373,29 @@ print(j)
 DT[, j, env = list(j = j)]
 ```
 
+### Injecting functions using strings in `env`
```

Comment:
Where does the term "inject" come from?
I think the more typical R term would be "substitute" (like the function of that name)